### DISCUSSION TITLE
How to create psd file with alpha cannel

### DISCUSSION THREAD

**Discussion by bashi2020** [ImageMagick/ImageMagick#2429] [Help] [answered]
hey,guys
I want to create psd file with alpha channel.
But  I can not it.
I could to create tga file with alpha channel.
〇　convert -size 128x128 xc:#FFFFFF00  test.tga
But psd is transparent image no alpha channel.
×　convert -size 128x128 xc:#FFFFFF00  test.psd

And xc:＝achromatic color(ex:#22222200) psd is grayscale.
I want to know achromatic color psd convert to RGB mode.
I could to convert tga.
〇　convert test.tga -colorspace sRGB -type truecolor test.tga
But I can not to convert psd
×　convert test.psd -colorspace sRGB -type truecolor test.psd
Sorry for my English is bad.
Thank you.
![2020-08-18_21h02_37](https://user-images.githubusercontent.com/69849758/90511738-e7978680-e197-11ea-8a72-6297b5555a67.jpg)
![2020-08-18_21h05_17](https://user-images.githubusercontent.com/69849758/90511752-ecf4d100-e197-11ea-89e9-ab53e3434c26.jpg)

**Comment by fmw42** [comment, marked as answer]
ImageMagick currently puts transparency into the background transparency and not the alpha channel. A developer might comment further. I suspect a new define would be needed to specify whether the transparency should go to the alpha channel.

**Reply by bashi2020** [reply]
Thank you for your reply.
I didn't know it.
Thank you so much.
